ilastik/shell/gui/ilastikShell.py

```diff
@@ -937,7 +937,26 @@ def _createSettingsMenu(self):
         def editShortcuts():
             mgrDlg = ShortcutManagerDlg(self)
 
+        def openConfigFolder():
+            from ilastik.utility import filemanager
+            from volumina.utility import preferences
+
+            path = preferences.get_path().parent
+            msg = None
+
+            try:
+                filemanager.open(path)
+            except filemanager.MissingProgramError as e:
+                msg = f"<p>Program <tt>{e.name}</tt> is not installed</p>"
+            except Exception as e:
+                msg = f"<p>Failed to open the default file manager:<br><tt>{e}</tt></p>"
+
+            if msg:
+                msg += f"<p>Open the following config folder manually:<br><tt>{path}</tt></p>"
+                QMessageBox.critical(self, "Config Folder Error", msg)
+
         menu.addAction("&Keyboard Shortcuts").triggered.connect(editShortcuts)
+        menu.addAction("Open &Config Folder...").triggered.connect(openConfigFolder)
```

Comment:
to be honest, I don't think that it is that important to have it in the gui.
However, now that you have added the niceness to open file locations in native file browsers, we can finally easily add the "navigate to log file" button, which will be very useful :)

Reply:
What do you think about moving our log files under the corresponding `appdirs` directory too? `ilastik_log.txt` in the home dir is slightly annoying for me, but on the other hand I see that the log file is much more important for users to locate.

Reply:
sure for our users this is not of much importance, as they (hopefully) never have to look at it, I think for devs it would be nice to be able to have this location configurable.